modules/ROOT/pages/create-reference-map.adoc

```diff
@@ -1,10 +1,10 @@
 = Creating Reference Maps for Message Types
 
-When you create a new Source at Host message type or a Receive from Partners message type formatted with JSON or XML, you must associate the message type with a DataWeave reference map. This is optional for Receive from Partners message types formatted EDFIACT or X12. Reference maps can include custom message attributes that get extracted from the message payloads at runtime. The extracted attributes are visible in the payload and are available for searching and tracking.
+When you create a new Source at Host or Receive from Partners message type with a CSV, JSON, or XML format, you must associate the message type with a DataWeave reference map. This is optional for Receive from Partners message types with an EDFIACT or X12 format. Reference maps can include custom message attributes that get extracted from the message payloads at runtime. The extracted attributes are visible in the payload and are available for searching and tracking.
 
-The input of a reference map is the payload that follows the schema uploaded in the definition for the message type.
+If the definition for the message type includes an uploaded schema, the input of the reference map is the payload that follows the schema.
```

Comment:
Any way to be active? 
The payload takes the reference map... as input?

Reply:
Can't think of a way to do this.